mifospay/src/main/java/org/mifospay/notification/ui/NotificationScreen.kt

```diff
@@ -0,0 +1,182 @@
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.mifospay.core.model.domain.NotificationPayload
+import org.mifospay.R
+import org.mifospay.core.designsystem.component.MfLoadingWheel
+import org.mifospay.core.designsystem.component.MifosTopAppBar
+import org.mifospay.core.designsystem.theme.MifosTheme
+import org.mifospay.core.designsystem.theme.historyItemTextStyle
+import org.mifospay.core.designsystem.theme.styleMedium16sp
+import org.mifospay.core.ui.EmptyContentScreen
+import org.mifospay.notification.presenter.NotificationUiState
+import org.mifospay.notification.presenter.NotificationViewModel
+
+@Composable
+fun NotificationScreen(viewmodel: NotificationViewModel = hiltViewModel()) {
+    val uiState by viewmodel.notificationUiState.collectAsStateWithLifecycle()
+    val isRefreshing by viewmodel.isRefreshing.collectAsStateWithLifecycle()
+    NotificationScreen(
+        uiState = uiState,
+        isRefreshing = isRefreshing,
+        onRefresh = {
+            viewmodel.refresh()
+        }
+    )
+}
+
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalMaterialApi::class)
+@Composable
+fun NotificationScreen(
+    uiState: NotificationUiState,
+    isRefreshing: Boolean,
+    onRefresh: () -> Unit
+) {
+    val pullRefreshState = rememberPullRefreshState(isRefreshing, onRefresh)
+    Box(Modifier.pullRefresh(pullRefreshState)) {
+        Column(
+            modifier = Modifier.fillMaxSize()
+        ) {
+            MifosTopAppBar(titleRes = R.string.notifications)
+            when (uiState) {
+                is NotificationUiState.Error -> {
+                    EmptyContentScreen(
+                        modifier = Modifier,
+                        title = stringResource(id = R.string.error_oops),
+                        subTitle = stringResource(id = R.string.unexpected_error_subtitle),
+                        iconTint = Color.Black,
+                        iconImageVector = Icons.Rounded.Info
+                    )
+                }
+
+                NotificationUiState.Loading -> {
+                    MfLoadingWheel(
+                        contentDesc = stringResource(R.string.loading),
+                        backgroundColor = Color.White
+                    )
+                }
+
+                is NotificationUiState.Success -> {
+                    if (uiState.notificationList.isEmpty()) {
+                        EmptyContentScreen(
+                            modifier = Modifier,
+                            title = stringResource(id = R.string.error_oops),
```

Comment:
Text is wrong, It should be something like 
title - Nothing to Notify
subtitle - There is nothing to show